easypost/__init__.py

```diff
@@ -217,8 +190,6 @@ def encode_none(cls, out, key, value):
     def _encode_inner(cls, params):
```

Comment:
I don't think the `_encode_inner` is being called anymore, is that so? If not, we'll want to remove it; however, it was calling the encoder for `datetime`. Is that now getting properly encoded? Do we have a test for that? If so and `encode_datetime` isn't needed anymore, we could also remove that.

Similarly, I don't believe that `def encode` is being used anymore either and if so could be removed along with the `urlencode` function from `six`.

Reply:
> I don't think the _encode_inner is being called anymore, is that so?

It still does for GET requests via build_url -> encode.

> it was calling the encoder for datetime. Is that now getting properly encoded?

I only saw datetime arguments on GET requests, so those should still be ok since it's still being used there.

> Similarly, I don't believe that def encode is being used anymore either

See previous.

Reply:
Ah perfect, must have just missed one of the calls. Thanks.